lib/brewdler/brew_installer.rb

```diff
@@ -2,14 +2,14 @@ module Brewdler
   class BrewInstaller
     def self.install(name, options = {})
       if `which brew`; $?.success?
-        command = "brew install #{name}"
+        command = [name]
         unless options[:args].nil?
           options[:args].each do |arg|
-            command << " --#{arg}"
+            command << "--#{arg}"
           end
         end
 
-        `#{command}`
+        system "brew", "install", *command
```

Comment:
OK. One thing is what is the easiest way to hide the output? `system "brew", "install", "foo", "&>/dev/null"` doesn't work. Only `system "brew install foo &>/dev/null"` works. But I don't want to use the latter one as this PR's whole point is to avoid shell injection(by using list to pass command).

Second thing, should I hide stdout only(like the current behavior) or hide both stdout and stderr(which is little strange, as it will hide errors in the mean time.)?